.gitignore

```diff
@@ -1,5 +1,5 @@
 # NPM/Node
-/node_modules/
+node_modules/*
```

Comment:
Just adding our discussion from earlier. If we can package up the typedef into our own module (or even a GitHub repo) and do an npm install instead of having it & other content checked into source then I think that would be better.

All depends on the amount of effort for that vs. getting that PR into the definitelytyped repo (assuming they publish to npmjs fairly quickly)